server/src/main/java/org/elasticsearch/index/IndexVersion.java

```diff
@@ -117,12 +116,36 @@ private static IndexVersion registerIndexVersion(int id, Version luceneVersion,
     public static final IndexVersion V_8_9_1 = registerIndexVersion(8_09_01_99, Version.LUCENE_9_7_0, "955a80ac-f70c-40a5-9399-1d8a1e5d342d");
     public static final IndexVersion V_8_10_0 = registerIndexVersion(8_10_00_99, Version.LUCENE_9_7_0, "2e107286-12ad-4c51-9a6f-f8943663b6e7");
     public static final IndexVersion V_8_11_0 = registerIndexVersion(8_11_00_99, Version.LUCENE_9_7_0, "f08382c0-06ab-41f4-a56a-cf5397275627");
+
     /*
-     * READ THE JAVADOC ABOVE BEFORE ADDING NEW INDEX VERSIONS
+     * READ THE COMMENT BELOW THIS BLOCK OF DECLARATIONS BEFORE ADDING NEW INDEX VERSIONS
      * Detached index versions added below here.
      */
+    public static final IndexVersion V_8_500_000 = registerIndexVersion(8_500_000, Version.LUCENE_9_7_0, "bf656f5e-5808-4eee-bf8a-e2bf6736ff55");
```

Comment:
Are we going to switch to a similar pattern as for `TransportVersion` where we use descriptive IDs?

Reply:
Yes, this is going to be changed soon to match TransportVersion. Don't have a timescale for that yet though.